_posts/2016-07-07-sprint-demo-4-july-7-2016.md

```diff
@@ -15,8 +15,5 @@ See some of the fine features that are going into Pulp 2.10. Below is the agenda
   * GPG signature on RPMs (27:07)
   * RPM rsync distributor (31:02)
 
-&nbsp;
```

Comment:
They all look fine, but I did think it was a little odd that you removed this line, but also add it in Sprint 9. I think I prefer without the extra space, but I'll leave it to you. Consider this an optional change.